web/src/components/filings-calendar/tax-access-modal/TaxAccessModal.test.tsx

```diff
@@ -36,6 +40,7 @@ describe("<TaxAccessModal />", () => {
         generateUserData({
           profileData: generateProfileData({
             businessPersona: persona as BusinessPersona,
+            legalStructureId: randomLegalStructure().id,
```

Comment:
This wouldn't be necessary if we hadnt set it to undefined in the generators

I think it's the reasonable assumption that the generators set a value in pretty much every field

Reply:
agree will remove